frontend/kubecloud/src/views/SignUpView.vue

```diff
@@ -23,7 +23,7 @@
           prepend-inner-icon="mdi-email"
           variant="outlined"
           class="auth-field"
-          :error-messages="errors.email"
+          :rules="[RULES.email]"
```

Comment:
kassemo@inc_ubaid.com this failed from backend in validaiton 
{
    "status": 400,
    "message": "Validation failed",
    "error": "Field 'Email' failed validation on 'email'\n"
}

Reply:
same on sign in